teamB/shinInHuck/myapp/package.json

```
@@ -14,12 +14,10 @@
     "cors": "^2.8.5",
     "dbmate": "^1.0.3",
     "dotenv": "^16.0.3",
-    "express": "^4.18.2",
+    "express": "^4.18.1",
     "morgan": "^1.10.0",
     "mysql": "^2.18.1",
+    "mysql2": "^2.3.3",
```

해당 부분은 같은 기능을 하기 때문에 npm 을 통해 하나를 삭제해 주셔야합니다
`npm uninstall mysql` 을 사용해 해당 모듈을 삭제해주세요 
```suggestion
    "mysql2": "^2.3.3",
```